javascript/src/test/components/BasicCourseSearch/CourseFilters.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from "@testing-library/react";
+import CourseFilters from "main/components/BasicCourseSearch/CourseFilters";
+
+
+describe("CourseFilters tests", () => {
+
+    const cancelled = false;
```

Comment:
You should also test whether the callback functions are working properly. So one way to do this is by maybe having a state object and you test if the callback functions are properly changing the state object.

```
const mockState = {
    cancelled: false,
    closed: false,
    full: false
};

test("cancel callback function modifies state", () => {
// render component
// find checkbox component in DOM
// use userEvents to click on the checkbox
// check for state to update
})
```

Let me know if you have any questions.

Reply:
May be more useful if the test was done for the higher level component that uses this one.